tests/client/disco/components/TestInstallButton.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { Simulate, renderIntoDocument } from 'react-addons-test-utils';
+
+import InstallButton from 'disco/components/InstallButton';
+import {
+  DOWNLOADING,
+  INSTALLED,
+  INSTALLING,
+  UNINSTALLED,
+  UNINSTALLING,
+  UNKNOWN,
+} from 'disco/constants';
+
+
+describe('<InstallButton />', () => {
+  function renderButton(props) {
+    return renderIntoDocument(<InstallButton { ...props } />);
+  }
+
+  it('should not be disabled if addonState is UNINSTALLED', () => {
+    const button = renderButton({addonState: UNINSTALLED});
+    const root = findDOMNode(button);
+    const checkbox = root.querySelector('input[type=checkbox]');
+    assert.equal(checkbox.hasAttribute('disabled'), false);
+  });
+
+  it('should be disabled if isDisabled addonState is UNKNOWN', () => {
+    const button = renderButton({addonState: UNKNOWN});
+    const root = findDOMNode(button);
+    const checkbox = root.querySelector('input[type=checkbox]');
+    assert.equal(checkbox.hasAttribute('disabled'), true);
+  });
+
+  it('should not be shown as installed if addonState is UNINSTALLED', () => {
+    const button = renderButton({addonState: UNINSTALLED});
+    const root = findDOMNode(button);
+    const checkbox = root.querySelector('input[type=checkbox]');
+    assert.equal(checkbox.checked, false);
+    assert.notInclude(root.getAttribute('class'), 'installed');
```

Comment:
I think these would be better off using `assert(!root.classList.contains('installed'))` since if this had an `uninstalled` or `not-installed` or whatever class this would fail.